openhands/llm/llm.py

```diff
@@ -2,7 +2,7 @@
 import copy
 import warnings
 from functools import partial
-from typing import Union
+from typing import Any, Dict, Union
```

Comment:
```suggestion
from typing import Any, Union
```

We can use dict since it's python >3.10

Reply:
mypy disagrees with you, though

Reply:
Fixed.